core/modules/catalog/helpers/variant/isOptionAvailable.ts

```diff
@@ -3,7 +3,7 @@ import findConfigurableVariant from './findConfigurableVariant'
 /**
  * Checks if variant with specific configuration exist
  */
-export default function isOptionAvailable (context, { product, configuration }): boolean {
-  const variant = findConfigurableVariant({ product: product, configuration: configuration, availabilityCheck: true })
+export default function isOptionAvailable (context, { product, configuration, fullMatch = false }): boolean {
+  const variant = findConfigurableVariant({ product: product, configuration: configuration, availabilityCheck: true, fullMatch: fullMatch })
```

Comment:
can you change it?
```
const variant = findConfigurableVariant({
  product,
  configuration,
  availabilityCheck: true,
  fullMatch
})
```

Reply:
Or maybe even?:
```js
const variant = findConfigurableVariant({
  product,
  configuration,
  fullMatch,
  availabilityCheck: true
})
```

To elegantly separate properties with the provided value from shortened ones